ironfish/src/account/accounts.ts

```diff
@@ -135,11 +135,12 @@ export class Accounts {
       return
     }
 
-    this.updateHeadState = new ScanState()
+    const scan = new ScanState()
```

Comment:
We cannot reference this.updateHeadState because it can be cleared by shutdown() and this causes a deadlock.